README.md

```diff
@@ -187,6 +187,18 @@ Note that only `method`, `headers`, `redirect` and `body` are allowed in `window
 }
 ```
 
+#### Default Headers
+
+If no values are set, the following request headers will be sent automatically:
+
+Header            | Value                                                   
+----------------- | --------------------------------------------------------
+`Accept-Encoding` | `gzip,deflate`                                          
+`Accept`          | `*/*`                                                   
+`Connection`      | `close`                                                 
```

Comment:
@ahmadnassri, no, the default only implies that it is set automatically in all cases when the header is not explicitly set. This header is special in that it is set automatically, when the header isn't explicitly set **and** `agent` isn't set.